android/MobileSdk/build.gradle.kts

```
@@ -145,4 +147,18 @@ dependencies {
     testImplementation("org.json:json:20230618")
     androidTestImplementation("com.android.support.test:runner:1.0.2")
     androidTestImplementation("com.android.support.test.espresso:espresso-core:3.0.2")
+
+    // DC-API dependencies
+    implementation("androidx.credentials:credentials:1.5.0")
+    implementation("androidx.credentials.registry:registry-digitalcredentials-preview:1.0.0-alpha01")
+    implementation("androidx.credentials.registry:registry-digitalcredentials-mdoc:1.0.0-alpha01")
+    implementation("androidx.credentials.registry:registry-provider:1.0.0-alpha01")
+    implementation("androidx.credentials.registry:registry-provider-play-services:1.0.0-alpha01")
```

Not sure if this makes sense here, but looks like the version should always be the same

```suggestion
    val androixCredentialsRegistryVersion = "1.0.0-alpha01"
    implementation("androidx.credentials.registry:registry-digitalcredentials-preview:$androixCredentialsRegistryVersion")
    implementation("androidx.credentials.registry:registry-digitalcredentials-mdoc:$androixCredentialsRegistryVersion")
    implementation("androidx.credentials.registry:registry-provider:$androixCredentialsRegistryVersion")
    implementation("androidx.credentials.registry:registry-provider-play-services:$androixCredentialsRegistryVersion")
```